app/code/Magento/GraphQl/etc/schema.graphqls

```diff
@@ -25,9 +25,20 @@ input FilterTypeInput @doc(description: "FilterTypeInput specifies which action
 type SearchResultPageInfo @doc(description: "SearchResultPageInfo provides navigation for the query response") {
     page_size: Int @doc(description: "Specifies the maximum number of items to return")
     current_page: Int @doc(description: "Specifies which page of results to return")
+    sort_fields: SortFields @doc(description: "An object that includes the default sort field and all available sort fields") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Category\\SortFields")
```

Comment:
`sort_fields` should be moved to `Products` type, as mentioned in the issue description, because `SearchResultPageInfo` is meant for storing paginating information.

Reply:
Okey